src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -666,7 +666,7 @@ protected virtual void RequestRun()
         /// </summary>
         protected virtual void NodeModified(NodeModel node)
         {
-
+            HasUnsavedChanges = true;
```

Comment:
Suppose we add a new node type or someone removes that `base.NodeModified(node)` call?  I won't beleaguer this point, though.  LGTM.